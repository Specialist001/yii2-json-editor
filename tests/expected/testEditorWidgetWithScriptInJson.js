dataJsonEditor_729ee6af = new JSONEditor(document.getElementById('data-json-editor'), {"mode":"tree","onChange":function() {jQuery('#data').val(dataJsonEditor_729ee6af.getText());}}, {"script":"\u003Cscript\u003Ealert(\u0022XSS\u0022);\u003C\/script\u003E"});
jQuery('#data').parents('form').submit(function() {jQuery('#data').val(dataJsonEditor_729ee6af.getText());});