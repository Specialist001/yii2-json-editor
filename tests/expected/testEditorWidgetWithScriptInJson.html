<input type="hidden" id="data" name="data" value="{&quot;script&quot;: &quot;&lt;script&gt;alert(\&quot;XSS\&quot;);&lt;/script&gt;&quot;}" data-json-editor-name="dataJsonEditor_729ee6af"><div id="data-json-editor"></div>